packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -688,8 +688,14 @@ contract BridgeFacet is BaseConnextFacet {
         // balance read about it
 
         uint256 starting = IERC20(_asset).balanceOf(address(this));
+        uint256 liquidityFee = _getFastTransferAmount(
+          _args.amount,
+          (s.LIQUIDITY_FEE_DENOMINATOR - s.LIQUIDITY_FEE_NUMERATOR),
+          s.LIQUIDITY_FEE_DENOMINATOR
+        );
```

Comment:
maybe we should rename this utility to `muldiv` or something. as it reads now, its a bit confusing and this is the whole implementation:

```ts
/**
   * @notice Calculates fast transfer amount.
   * @param _amount Transfer amount
   * @param _liquidityFeeNum Liquidity fee numerator
   * @param _liquidityFeeDen Liquidity fee denominator
   */
  function _getFastTransferAmount(
    uint256 _amount,
    uint256 _liquidityFeeNum,
    uint256 _liquidityFeeDen
  ) private pure returns (uint256) {
    return (_amount * _liquidityFeeNum) / _liquidityFeeDen;
  }
```